File: Module_11/ACP/4.py
# ==================================
# POWER OF 8
# ==================================

n = int(input("Enter a number: "))

while n > 1 and n % 8 == 0:
    n = n // 8

if n == 1:
    print("Power of 8")
else:
    print("Not a Power of 8")

File: Module_24/ACP/4.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        try {

            System.out.print("Enter first number: ");
            int a = sc.nextInt();

            System.out.print("Enter second number: ");
            int b = sc.nextInt();

            int result = a / b;

            int[] arr = {10, 20, 30};

            System.out.println("Array Element: " + arr[5]);

            System.out.println("Result: " + result);

        }
        catch (ArithmeticException | ArrayIndexOutOfBoundsException e) {

            System.out.println("Exception Caught: " + e);
        }

        finally {

            System.out.println("Program Finished.");
            sc.close();
        }
    }
}